app/models/logs_search.rb

```diff
@@ -22,16 +21,24 @@ def validate_term
     end
   end
 
+  def validate_presence
+    if term.length == 0
+      self.errors.add(:search_term, 'cannot be empty')
+    end
+  end
```

Comment:
I tried to use the `ActiveRecord` type I18n validations in `en.yml` here but I couldn't get anything to work. 
- I tried including `ActiveMode::Translations`, 
- I also tried using the `activemodel:` key in the `en.yml` file, 
- I also tried adding a message directly to `validate :term, presence: { message: "Hello"}`

I couldn't get any of that to work correctly so I went for a custom `validate_presence` method here in the model.

Reply:
It actually works better this way as it means we avoid the duplicate error summary eg:
<img width="1067" alt="screenshot 2019-01-17 at 17 41 03" src="https://user-images.githubusercontent.com/2160769/51296040-16f78600-1a7f-11e9-905f-ae2f633ea9d9.png">

Reply:
I'm fine with this - it might surprise a reader a little ("why didn't they use the standard validations here?"), but it's not unclear what's going on.

The bonus single error message in the summary is nice, too.